### DISCUSSION TITLE
New laser configuration to the makerbase mks dlc v2.0

### DISCUSSION THREAD

**Discussion by baselbaransi** [arkypita/LaserGRBL#1838] [Q&A]
Hi there, 
I bought a new laser with 3pin cable, and having trouble to make it work with the board i have the makerbase mks dlc v2.0.
This is the board on the laser
![Screenshot_20220418-125645](https://user-images.githubusercontent.com/57457668/167327847-1c54a29f-f6fd-481f-838b-5225a4e32b85.jpg)
 The seller recommended to connect it like that, but it didn't work, any ideas on how to make it work?
Or maybe need other software for engraving instead of gprl laser.
Thanks in advance

**Comment by StuartB4** [comment]
I think it should be like this picture. The 12v + and - pins supply the power and the S and G are the PWM and Ground.
If that does'nt work connect the G and the - of the 12v pins together and the Yellow on the S pin. 

![image](https://user-images.githubusercontent.com/16980026/167331018-3498835c-74c2-4af4-a8af-f6811ffca815.png)

**Comment by StuartB4** [comment]
If you use the pins marked Laser you will have no control over the power to the laser. Thats what the S (PWM) pin does.
But you need a permanent 12v supply, that is what the 12V+/- pins do.

**Comment by baselbaransi** [comment]
Thanks dears, i will try this option
